SConstruct

```diff
@@ -251,8 +257,10 @@ if env_base["target"] == "debug":
     # http://scons.org/doc/production/HTML/scons-user/ch06s04.html
     env_base.SetOption("implicit_cache", 1)
 
-if not env_base["tools"]:
-    # Export templates can't run unit test tool.
+if env_base["dev"] or env_base["tools"]:
+    env_base["tests"] = True
```

Comment:
Why this change? `tests` should not be enabled by default with `tools`. The previous logic was fine.

Reply:
I was previously concerned about these changes being important to the PR, but since you disapproved of them, I ended up reverting them.